src/data_source/storage/sql.rs

```diff
@@ -3183,6 +3183,11 @@ pub mod testing {
                 value BYTEA  NOT NULL UNIQUE
             );
         
+
+            ALTER TABLE header
```

Comment:
Could just include this in the `CREATE TABLE` statement

Reply:
dont we need a separate migration for this?

Reply:
this is for test merkle tree so didn't want to add it in migration